### PR TITLE
Ensure package check uses the latest version of packages

### PR DESCRIPTION
**Pull request by fabricioferreira**
- Fixed the package version check by ensuring the version list returned from NuGet is ordered and the latest version is picked up for comparison
- Improved the local development experience by adding additional steps to the build.bat file

### CODE REVIEW

**Comment by fabricioferreira** [comment]
This PR fixes the issue #35

File: build.bat [hunk 1]
```diff
@@ -1 +1,9 @@
+cd ./buildtab
+cmd /c "npm i && npm run build"
```
Comment (chwebdude) [inline build.bat]: Why not `yarn` (which is used for the resto of the setup)?

Comment (fabricioferreira) [reply build.bat]: No particular reason. I will change it to use `yarn` as well.

Comment (fabricioferreira) [reply build.bat]: Done.

**Pull request merged**